components/common/Footer.tsx

```diff
@@ -7,7 +7,8 @@ import {
   Text,
   useColorModeValue,
 } from "@interchain-ui/react";
-import { dependencies, products, Project } from "@/config";
+import { Project, dependencies, products } from "../../config";
+// import { dependencies, products, Project } from "@/config";
```

Comment:
you don't need this change, just do it.
![image](https://github.com/agoric-labs/hardened-create-cosmos-app/assets/96726225/ea7768fd-720b-4cc2-a549-117564cb1322)

Reply:
I'm trying to figure out how to run it in a turborepo, your solution gimme some idea, thanks.